### ISSUE TITLE
FR: (BETTER) Room search

### ISSUE THREAD

**Issue by JuniorJPDJ** [Nheko-Reborn/nheko#316]
Hi guys,

I'd like to filter/search for rooms by name.

It's one of last features I miss so much.

**Comment by deepbluev7** [comment]
Have you tried Ctrl+K? Can you explain, why that doesn't work for you? (I'm not opposed to your proposal, just want to figure out, where the differences in behaviour should be.)

**Comment by JuniorJPDJ** [comment]
I haven't tried it and I haven't seen it documented anywhere.
My bad. That's exactly what I was looking for.
What do you think about making it visible whole time like on telegram?
It would be much more intuitive.
![Screenshot_20201108_151557](https://user-images.githubusercontent.com/7334549/98467395-50116500-21d5-11eb-8960-6f069d9dcf8d.png)

**Comment by JuniorJPDJ** [comment]
Also search results could/should be sorted by activity, just like main room select panel.

**Comment by deepbluev7** [comment]
Yeah, the existing search algorithm needs some improvements. I think someone was working on that too. Apart from that, yes, we need to hint at it better, many people can't find that shortcut. Just making it always visible would change the interaction pattern though. I use that to quickly jump to a room, if it was always visible it would work more like a filter and Enter to accept would be confusing. But that will be part of the UI enhancements, once some more low level stuff is done.

**Comment by JuniorJPDJ** [comment]
Telegram have it done like this:
It's a filter, but when you accept result you are interested in - filter is cleared and you are just scrolled to position of room on whole list.
Also shortcut could make this inputbox active instead of drawing another list.
IMHO It would satisfy your usecase and would be muuuch more userfriendly.
Telegram Desktop is best UX designed IM I ever used so It could be cool to "steal" some solutions from it :D and this is one of features/solutions I personally very like in it.
Most of normal users still prefer to use mouse than hotkeys.

**Comment by JuniorJPDJ** [comment]
I changed title and will reopen it as UX improvement issue.

**Comment by deepbluev7** [comment]
Duplicate of #81
